src/HRInPocket/Services/HRInPocket.Parsing.hh.ru/HRInPocket.Parsing.hh.ru/Service/ParsehhService.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Linq;
+
+using AngleSharp;
+
+using HRInPocket.Parsing.hh.ru.Interfaces;
+using HRInPocket.Parsing.hh.ru.Models.Entites;
+
+namespace HRInPocket.Parsing.hh.ru.Service
+{
+    public class ParsehhService : IParsehhService
+    {
+        public IParsehh GetPasrse()
+        {
+            return new Parsehh();
+        }
+    }
+    /// <summary>
+    /// Парсер загружает страницу по адресу https://hh.ru/search/vacancy
+    /// </summary>
+    public class Parsehh : IParsehh
+    {
+        public event Action<Vacancy> Result;
+        readonly string HHUrl = "https://hh.ru/search/vacancy";
+
+        /// <summary>
+        /// Парсит https://hh.ru/search/vacancy и возвращает значения по готовности через событие Result
+        /// </summary>
+        /// <param name="GetParameters">
+        /// Если нужно задать точные параметры поиска, передайте их в свойстве GetParameters
+        /// в формате "?param1=value&param2=value&...&paramN=value"
+        /// </param>
+        public async void Parse(string GetParameters=null)
```

Comment:
Нужно переделать метод так, чтобы он возвращал задачу
```suggestion
        public async Task Parse(string GetParameters=null)
```

Reply:
Принято